File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.py
def solution(my_string):
    answer = 0
    temp = ""
    
    for ch in my_string:
        if ch.isdigit():
            temp += ch
        else:
            if temp:
                answer += int(temp)
                temp = ""
    
    if temp:
        answer += int(temp)
        
    print(answer)
    
    return answer

File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.swift
import Foundation

func solution(_ my_string:String) -> Int {
    var temp: String = ""
    var answer: Int = 0
    
    for ch in my_string {
        if ch.isNumber {
            temp.append(ch)
            print(temp)
        } else {
            if !temp.isEmpty {
                answer += Int(temp)!
                temp = ""
            }
        }
    }
    
    if !temp.isEmpty {
        answer += Int(temp)!
    }
    
    print(answer)
    
    return answer
}